expenses-manager-frontend/src/main/frontend/src/app/login/login.component.html

```diff
@@ -5,6 +5,9 @@ <h1 class="logo-text">Expenses Manager</h1>
     </div>
 
     <form class="login-form">
+        <div class="error">
```

Comment:
In loc sa faci tot acest workaround cred ca era suficient sa ii setezi la input atributul required si va sari validarea browserului